mimic/canned_responses/maas_test_check.py

```diff
@@ -0,0 +1,67 @@
+"""
+Canned response for /test-check
+"""
+
+import collections
+
+
+def test_check(checkType):
+    """
+    Support test-check API
+    """
+    test_check_responses = collections.defaultdict(lambda: "")
+    test_check_responses["agent.cpu"] = [
+        {
+            "timestamp": 0,
+            "available": True,
+            "status": "success",
+            "metrics": {
+                "user_percent_average": {
+                    "data": 7.1695253641707,
```

Comment:
I won't pretend to understand all of the hard-coded values in any of my mimic PRs either. I just know that when I ask the real production endpoints from a real rackspace-cloud server with a real agent, that's what is returned to me and it makes the cloud-intelligence UI happy. I don't want random stuff that may or may not be valid.